openquake/cat/completeness/generate.py

```diff
@@ -56,16 +57,29 @@ def get_completenesses(fname_config, folder_out):
     min_mag_compl = config[key].get('min_mag_compl', None)
     apriori_conditions = config[key].get('apriori_conditions', {})
     cref = config[key].get('completeness_ref', None)
-    step = config[key].get('step', 8)
     mrange = config['completeness'].get('deviation', 1.0) 
-
-    _get_completenesses(mags, years, folder_out, num_steps, min_mag_compl,
-                        apriori_conditions, cref, step, mrange)
+    try:
+        _n_vals_per_iter = config[key]['step']
+        logging.warning(
+            "`step` parameter deprecated. Use `_n_vals_per_iter` instead."
+        )
+    except KeyError:
+        _n_vals_per_iter = 8
+
+    _get_completenesses(mags, 
+                        years, 
+                        folder_out=folder_out, 
+                        num_steps=num_steps,
+                        min_mag_compl=mini_mag_compl,
+                        apriori_conditions=apriori_conditions,
+                        cref=cref,
+                        _n_vals_per_iter=_n_vals_per_iter,
+                        mrange=mrange)
 
 
 def _get_completenesses(mags, years, folder_out=None, num_steps=0,
                         min_mag_compl=None, apriori_conditions={},
-                        completeness_ref=None, step=6, mrange=1.0):
+                        completeness_ref=None, _n_vals_per_iter=6, mrange=1.0):
```

Comment:
I'm not sure if `_get_completenesses` is called anywhere else, but it may not be necessary to set as a key word arg if we can't define in the config. here we have default 6 but above default is 8, maybe should be consistent if we don't have a reason